src/zenml/logging/step_logging.py

```diff
@@ -170,7 +170,14 @@ def __init__(self, logs_uri: str) -> None:
         Args:
             logs_uri: the URI of the logs file.
         """
-        self.storage = StepLogsStorage(logs_uri=logs_uri)
+        self.storage = StepLogsStorage(
+            logs_uri=logs_uri, max_messages=sys.maxsize
+        )
+        try:
+            with fileio.open(logs_uri, "r") as file:
```

Comment:
@adtygan this is the line the failing linting is talking about...

Reply:
As far as I understand, L177 is using `fileio` which has not been imported, and this is raising the error. Please correct me if I'm wrong.

Reply:
I have added the import now.

Reply:
I ran the command `bash scripts/lint.sh --fix` locally. In `step_logging.py` it swapped the order of imports, and seems to be fixing the Lint Check at https://github.com/zenml-io/zenml/actions/runs/8347170346/job/22846195340. But running this command has also changed a lot of other files. Should I still push the change to this branch?

Reply:
The script to run is `bash scripts/format.sh` to format the files. Not sure where the `--fix` came from, but you should undo any changes made by `bash scripts/lint.sh --fix` and run `bash scripts/format.sh` instead before repushing/committing.

Reply:
I ran the `format.sh` instead and it still made a lot of changes in other files. I pushed the commit and it has added a lot of commits here. I'm not sure if this is intended. It's my first time working on open-source and I apologise for any inconvenience.